### ISSUE TITLE
Typo in t.w.r.ErrorPage class docstring

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7797]
|[<img alt="twm's avatar" src="https://avatars.githubusercontent.com/u/43662?s=50" width="50" height="50">](https://github.com/twm)| @twm reported|
|-|-|
|Trac ID|trac#7797|
|Type|defect|
|Created|2015-02-24 21:16:33Z|
|Branch|https://github.com/twisted/twisted/tree/t.w.r.ErrorPage-class-docs-7797|

It uses @ivar where @type is intended, resulting in [botched API docs](https://twistedmatrix.com/documents/15.0.0/api/twisted.web.resource.ErrorPage.html):

```
Instance Variable 	brief 	A short string which will be included in the response body. (type: str)
Instance Variable 	detail 0 	A longer string which will be included in the response body.
Instance Variable 	detail 	str
```

Attachments:

* [7797.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/db1/db1e8b83ded5fd9e592f8e338f3e255c63cbeccf/ed78101db66d7438a2d157a21ecbc28d97bf4533.patch) (723 bytes) - added by twm on 2015-02-24 21:17:35Z - Patch for the typo

<details><summary>Searchable metadata</summary>

```
trac-id__7797 7797
type__defect defect
reporter__twm twm
priority__lowest lowest
milestone__ 
branch__branches_t_w_r_ErrorPage_class_docs_7797 branches/t.w.r.ErrorPage-class-docs-7797
branch_author__adiroiban adiroiban
status__closed closed
resolution__fixed fixed
component__web web
keywords__Easy Easy
time__1424812593501320 1424812593501320
changetime__1429459410995438 1429459410995438
version__None None
owner__twm twm

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="twm's avatar" src="https://avatars.githubusercontent.com/u/43662?s=50" width="50" height="50">](https://github.com/twm)<a name="note_2"></a>|@twm commented|
|-|-|

Patch for review: https://twistedmatrix.com/trac/attachment/ticket/7797/7797.patch

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @twm|
|-|-|

It looks good. Many thanks!

It also requires a news file for this patch as described here: http://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles

I am going to create a branch based on your patch, add the newsfile and send it to buildbot.

Will land if tests pass so there is no need to come back with a new patch.

Thanks again!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban commented|
|-|-|

(In [44501]) Branching to t.w.r.ErrorPage-class-docs-7797.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban commented|
|-|-|

Just some notes

```

$ ./bin/admin/build-apidocs . apidocs
$ chromium-browser apidocs/twisted.web.resource.ErrorPage.html 
```

we no longer have details 0

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_6"></a>|@adiroiban set status to `closed`|
|-|-|

(In [44503]) Merge t.w.r.ErrorPage-class-docs-7797: Fix apidoc docstring in t.w.r.ErrorPage.

Author: twm
Reviewers: adiroiban
Fixes: [#7797](https://github.com/twisted/twisted/issues/7797)
